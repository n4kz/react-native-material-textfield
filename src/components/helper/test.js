import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Helper from '.';

/* eslint-env jest */

const text = 'helper';

it('renders helper', () => {
  let helper = renderer
    .create(<Helper>{text}</Helper>)
    .toJSON();

  expect(helper)
    .toMatchSnapshot();
});
