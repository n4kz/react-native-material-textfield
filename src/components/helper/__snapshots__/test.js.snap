// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders helper 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "bottom": 0,
      "left": 0,
      "paddingVertical": 4,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    collapsable={undefined}
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
    helper
  </Text>
</View>
`;
